.github/workflows/doc-publish.yml

```diff
@@ -14,54 +14,45 @@ permissions:
   pull-requests: write
 
 jobs:
-  upload-docs:
```

Comment:
How about adding a comment on the top of the file that gives an overview of the URLs deployed to (and by what event)

Reply:
OK. I will do it tomorrow. It is late at night here :-)

Reply:
Done in https://github.com/sagemath/sage/pull/38220/commits/a74cc417cc292f00b447c3415b77e371e63e11cc

Reply:
Thanks.

Aren't we missing ways to refer to the latest livedoc?

Reply:
Good point. Let me think..

Reply:
One simple solution is to deploy twice for tag push: once to "doc-tag--sagemath.netlify.app" and once again to the production site "sagemath.netlify.app".

Reply:
It seems to be the only solution. As far as I see, netlify cli does not support multiple aliases.

Reply:
With the simple solution, we have choices to make. For second deploymen,t we may use

(1) production site:  "sagemath.netlify.app".
(2) or draft site: "doc-latest--sagemath.netlify.app"
(3) or draft site with another name "doc-...--sagemath.netlify.app"

I like (2). Which do you prefer?

Reply:
"doc-release" is not bad...

Reply:
"latest" typically means the latest _stable_ release

but where would the livedoc for the newest beta go?

Reply:
"doc-release". See https://github.com/sagemath/sage/pull/38220/commits/501573ed9081eaa70c51267dbbd090ff2ae244da

Reply:
It works: https://github.com/kwankyu/sage/actions/runs/9641539263